src/WebAPI/ModelsAPI.cs

```diff
@@ -540,8 +542,8 @@ public static async Task<JObject> EditModelMetadata(Session session,
             actualModel.Metadata.LoraDefaultWeight = lora_default_weight;
             actualModel.Metadata.LoraDefaultConfinement = lora_default_confinement;
             actualModel.Metadata.PredictionType = string.IsNullOrWhiteSpace(prediction_type) ? null : prediction_type;
+            handler.ResetMetadataFrom(actualModel);
```

Comment:
this is an actively dangerously line edit. Once again, _why are you editing code here at all?_